plugins/cisco_umbrella_enforcement/help.md

```diff
@@ -106,80 +125,87 @@ It accepts an array of JSON objects of the [Generic Event Format](https://docs.u
+|eventTime|string|None|True|The time the event was detected|None|2013-02-08T09:30:26Z|
+|eventType|string|None|False|Common name or classification of threat|None|severe|
+|externalURL|string|None|False|External page containing additional information about event|None|None|
+|fileHash|string|None|False|SHA-1 of file reported by appliance|None|02699626f388ed830012e5b787640e71c56d42d8|
+|fileName|string|None|False|Path to file exhibiting malicious behaviour|None|/path/to/file|
+|protocolVersion|string|https://example.com|True|The version of the protocol for the API|None|https://example.com|
+|providerName|string|Security Platform|True|The provider name for the API|None|Security Platform|
+|src|string|None|False|The first IP or hostname associated with the infected device|None|None|
 
 Example input:
 
 ```
-
-[{
-    "dstURL": "http://internetbadguys.bad-v5.com/a-bad-url-v1",
-    "alertTime": "2013-02-09T11:14:26.0Z",
-    "ID": "ba6a59f4-e692-4724-ba36-c28132c761de",
-    "deviceVersion": "13.7a",
-    "dstDomain": "internetbadguys.bad-v5.com",
-    "eventTime": "2013-02-09T09:30:26.0Z",
-    "protocolVersion": "1.0a",
-    "providerName": "Security Platform",
-    "disableDstSafeguards": true,
-    "eventHash": "e88b372b1f98882dca933fa8a2589670",
-    "fileName": "https://www.fuw.edu.pl/~rwys/pk/notatki_cl.txt",
-    "fileHash": "da89127fbe1d78313dbfff610b59ff24874bb983",
-    "externalURL": "https://www.fuw.edu.pl/~rwys/pk/notatki_cl.txt",
-    "src": "192.168.0.1",
-    "eventSeverity": "severe",
-    "eventType": "severe",
-    "eventDescription": "Some another threat"
-  },
-  {
-    "dstURL": "http://internetbadguys.bad-v6.com/a-bad-url-v2",
-    "alertTime": "2013-02-10T11:14:26.0Z",
-    "ID": "ba6a59f4-e692-4724-ba36-c28132c761de",
-    "deviceVersion": "13.7a",
-    "dstDomain": "internetbadguys.bad-v6.com",
-    "eventTime": "2013-02-10T09:30:26.0Z",
-    "protocolVersion": "1.0a",
-    "providerName": "Security Platform"
- }]
-
+{
+  "alertTime": "2013-02-08T09:30:26Z",
+  "deviceId": "12345678-1234-1234-1234-123456789123",
+  "deviceVersion": "1.0a",
+  "disableDstSafeguards": false,
+  "dstDomain": "www.internetbadguys.com",
+  "dstIP": "8.8.8.8",
+  "dstUrl": "http://internetbadguys.com/security?foo=there%20are%20spaces%20here",
+  "eventHash": "9de5069c5afe602b2ea0a04b66beb2c0",
+  "eventSeverity": "severe, bad, high",
+  "eventTime": "2013-02-08T09:30:26Z",
+  "eventType": "severe",
+  "fileHash": "02699626f388ed830012e5b787640e71c56d42d8",
+  "fileName": "/path/to/file",
+  "protocolVersion": "1.0a",
+  "providerName": "Security Platform"
+}
 ```
 
 ##### Output
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
-|id|string|True|Id of created event|
+|ID|object|True|List of added IDs|
 
 Example output:
 
 ```
-
 {
-  "ID": [
-    "42216a75,da20,4a1e,93bc-b07edfacc1f3"
-  ]
+  'ID': {'id': '5a050d19,1d08,4dd0,b8b4-6b34da9e4135'}
```

Comment:
It does not look like a list of added IDs

Reply:
It's called a list but it is actually an object, this was the original description 😄 I have fixed it up to look a bit better though

Reply:
Could you please add supported version to `plugin.spec.yaml`? 😄

Reply:
I just updated have a look 😄